lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -235,19 +235,11 @@ export class UserAgentApplication {
 
     this.redirectCallbacksSet = true;
 
-    const urlHash = window.location.hash;
-    const isCallback = this.isCallback(urlHash);
-
     // On the server 302 - Redirect, handle this
     if (!this.config.framework.isAngular) {
-      if (isCallback) {
-        this.handleAuthenticationResponse(urlHash);
-      }
-      else {
-        const pendingCallback = this.cacheStorage.getItem(Constants.urlHash);
-        if (pendingCallback) {
-          this.processCallBack(pendingCallback, null);
-        }
+      const pendingCallback = this.cacheStorage.getItem(Constants.urlHash);
```

Comment:
This will break the current flow for `loginSilent` as we no longer parsing the hash through `handleAuthenticationResponse`

Reply:
@step-lee Thanks for the review.

Do you mean acquireTokenSilent? We don't have any loginSilent APIs.

We are still parsing the hash in the constructor - this removes that logic from the handleRedirectCallbacks method. I removed the handleAuthenticationResponse call to avoid parsing the hash multiple times in the case of popup or iframe and the user mistakenly uses the handleRedirectCallbacks function.

Our code has evolved quite a bit, so the diff may be hiding some context for you. Feel free to ping me in Teams if you want to dive a bit deeper into this.

Reply:
Yes I mean the `acquireTokenSilent`. My apology on not reading the changes more carefully. You can discard my comment as this behavior is not happening in the constructor.